Tools/scripts/install-prereqs-ubuntu.sh

```diff
@@ -82,7 +82,7 @@ if apt-cache search python-wxgtk3.0 | grep wx; then
     SITL_PKGS+=" python-wxgtk3.0 libtool-bin"
 else
     # we only support back to trusty:
-    SITL_PKGS+=" python-wxgtk2.8"
+    SITL_PKGS+=" python-wxgtk2.8 realpath"
```

Comment:
This belongs in a separate check.  Look for what you actually want, not a specific version of Ubuntu :-)

`if apt-cache search realpath;`

I think we can just assume it is in coreutils if it's not a separate package.